src/main.cpp

```diff
@@ -1205,8 +1205,9 @@ int main(int Argc, char *Argv[]) {
 
     for (auto target : targets) {
         if (target == ISPCTarget::wasm_i32x4) {
-            Assert(targets.size() == 1 && "wasm32 supports only one target: i32x4");
```

Comment:
Why removing the assert? The text needs to be modified, but still there only one target for wasm: `wasm_i32x4`.

Reply:
you are right. that was a mistake.